README.md

```diff
@@ -1 +1,13 @@
-# ccd-definition-processor
\ No newline at end of file
+# ccd-definition-processor
+## Overview
+CI tooling to support colaberative development of CCD UI thought the use of text based representation of definitons. Supporting textual comparison and change control.
+## Usage
+```
+update options <sheet name> ...
```

Comment:
That isn't the command people use to run it, is it? Please provide working instruction of how to use the tool.